src/config/fcart.js

```diff
@@ -3,6 +3,13 @@ import { defineMessages } from 'react-intl';
 export default {
   gatewayUrl: 'http://localhost:8180/gateway/fcart',
 
+  filters: {
+    fields: {
+      objectProductionPlace: {
+        field: 'collectionobjects_common:objectProductionPlaceGroupList.objectProductionPlace.displayName',
+      },
+    }
```

Comment:
This just needs a comma at the end. `npm run lint` will complain, because the linter rules we use prefer a trailing comma everywhere. (And I need to get CI working again).